source/cfp.rst

```diff
@@ -0,0 +1,63 @@
+Call For Proposals
+==================
+
+We are planning to use `papercall.io`_ for handling CFP proposals.
+
+Daniele has set up a form with the following questions:
+
+Talk details
```

Comment:
Not every session will be a talk, but we have used "talk" throughout. Perhaps "session" would be less confusing?